index.d.ts

```diff
@@ -138,6 +138,15 @@ interface Tracer extends opentracing.Tracer {
    * LLM Observability SDK
    */
   llmobs: tracer.llmobs.LLMObs;
+
+  /**
+   * Baggage
```

Comment:
Could you please add some further description about the usage and link to https://opentelemetry.io/docs/concepts/signals/baggage/?